model_compression_toolkit/quantizers_infrastructure/pytorch/inferable_quantizers/activation_inferable_quantizers/activation_lut_pot_inferable_quantizer.py

```diff
@@ -40,7 +40,9 @@ def __init__(self,
                      num_bits: int,
                      cluster_centers: np.ndarray,
                      threshold: np.ndarray,
-                     signed: bool):
+                     signed: bool,
```

Comment:
keep the constants: 
multiplier_n_bits: int = MULTIPLIER_N_BITS,
eps: float = EPS